test/release/mathjs.sh

```diff
@@ -46,11 +46,6 @@ rm -rf tmp/mathjs \
 @@ -68 +75 @@ export function format (value, options) {
 -      return value.toString()
 +      return HACK(value).toString()
---- a/test/node-tests/treeShaking/treeShaking.test.js
-+++ b/test/node-tests/treeShaking/treeShaking.test.js
-@@ -35 +35 @@ describe('tree shaking', function () {
--  it('should apply tree-shaking when bundling', function (done) {
-+  if (0) it('should apply tree-shaking when bundling', function (done) {
```

Comment:
Test seems to work with `--annotations` in place:
https://github.com/mishoo/UglifyJS/runs/2092215139?check_suite_focus=true#step:3:8831
https://github.com/mishoo/UglifyJS/runs/2092215200?check_suite_focus=true#step:3:8829
https://github.com/mishoo/UglifyJS/runs/2092215257?check_suite_focus=true#step:3:8829

Reply:
Good stuff!